docs/deployment/engine_on_kubernetes.md

```diff
@@ -36,6 +36,19 @@ Spark on Kubernetes config master by using a special format.
 
 You can use cmd `kubectl cluster-info` to get api-server host and port.
 
+### Deploy Mode
+
+Kyuubi supports cluster and client deploy modes.
+
+To use client deploy mode specify:
+* spark.driver.host (headless service hostname of the pod you want to use as a driver)
+* spark.kubernetes.driver.pod.name (the name of the pod)
```

Comment:
Emm, why should we set it in client mode?

Reply:
no reason, just listing all the options here

Reply:
I can add a line that cluster mode is recommended, if there is no particular reason to use client
wdyt?

Reply:
IMO, the client mode is out-of-box. We just need to mention that it's Kyuubi also supports cluster mode and it's always recommended to use cluster mode on K8s, the minimum required configurations are ...

Reply:
updated